admin/js/admin.js

```diff
@@ -283,6 +283,17 @@
 
         if ( $( email_editor ).exists() ) {
 
+			/**
+			 * Due to a bug in WordPress 5.6+, the wp_editor instance has to be
```

Comment:
Same note here about that trac ticket. I could see it being helpful to list it here as well.

Reply:
Done.